src/coreneuron/io/phase2.cpp

```diff
@@ -1111,6 +1130,31 @@ void Phase2::populate(NrnThread& nt, const UserParams& userParams) {
                 pp->_tid = nt.id;
             }
         }
+
+        auto& r = tmls[itml].nmodlrandom;
+        if (r.size()) {
+            size_t ix{};
+            uint32_t n_randomvar = r[ix++];
+            assert(r.size() == 1 + n_randomvar + 5 * n_randomvar * n);
+            std::vector<uint32_t> indices{};
```

Comment:
nitpick: 

```suggestion
            std::vector<uint32_t> indices(n_randomvar);
```

Reply:
Ok. Will fix in next commit. I presume that starting out indices as correct size array means filling in in next loop with
```
indices[i] = r[ix++])
```
instead of pushback,